mon-pix/app/templates/components/comparison-window.hbs

```diff
@@ -1,9 +1,9 @@
 <PixModal @class="comparison-window-modal" @onClose={{@closeComparisonWindow}}>
 
-  <div class="pix-modal__close-link" aria-label={{t "common.actions.close"}} {{action (action @closeComparisonWindow)}}>
+  <button type="button" class="pix-modal__close-link" aria-label={{t "common.actions.close"}} {{action (action @closeComparisonWindow)}}>
     <span>{{t "common.actions.close"}}</span>
     <FaIcon @icon="times-circle" class="logged-user-menu__icon"></FaIcon>
```

Comment:
**suggestion**: un aria-label explique déjà la fonctionnalité sur le bouton qui encapsule cette icon. Il s'agit d'une icon décorative. Je propose de rajouter `aria-hidden="true"` (https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility)